zuul.d/ansible-security-jobs.yaml

```diff
@@ -224,3 +224,67 @@
     required-projects:
       - name: github.com/ansible/ansible
         override-checkout: stable-2.12
+- job:
+    name: ansible-ee-security-integration-qradar
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    parent: ansible-ee-tests
+    pre-run:
```

Comment:
I *think* something like this should be enough:

```
- playbooks/ansible-ee/integration/pre.yaml
- playbooks/ansible-network-appliance-base/pre.yaml
- playbooks/ansible-ee/qradar/pre.yaml
```

Note: playbooks/ansible-network-appliance-base/pre.yaml should be a copy of playbooks/ansible-security-qradar-appliance/pre.yaml c.f: https://github.com/ansible/ansible-zuul-jobs/pull/1395/files#r827313563